package.json

```diff
@@ -59,6 +59,7 @@
     "@types/react-tag-autocomplete": "^6.1.1",
     "autoprefixer": "^10.4.2",
     "del": "^6.0.0",
+    "dotenv": "^16.0.1",
```

Comment:
The reason I used this dependency is that Architect only uses its own mechanism to load the environment variable under sandbox (https://arc.codes/docs/en/reference/cli/sandbox). I don't have access to the environment variables through process.env when using the set plugin to expose the environment variables to the frontend. So I create a constant variable outside modules.exports to use it in both sandbox and set, but in order to read the environment variable (in this case the oidc port) I use dotenv. 

Other way to access the environment variables is using the `inventory` object like this: `inventory.inv._project.env.local`. This gives you an object with the environment variables for `testing`, `staging` and `production`. Like this:

`{
testing: {}, 
staging: {}, 
production: {}
}`

Now, locally all the environment variables from .env were loaded to `testing`. But from Architect Docs it says: 

> @env
Architect Sandbox will load variables for Sandbox’s current environment (testing, staging, or production) from a [local preferences file with @env](https://arc.codes/docs/en/reference/configuration/local-preferences#%40env). If a project contains a .env file, Architect will load those variables instead.

What I understood from this was that, for the current environment running, Architect loads the .env files and replaces it. But when I ran Architect using `ARC_ENV=staging` it still loaded it into `testing`.

I can implement it using the `inventory.inv._project.env.local.testing` object to access the environment variables and that way I don't need to use the dependency, but I was a little bit scared of using the environment variables hardcoded from `inventory.inv._project.env.local.testing`.

Reply:
I think that `inventory.inv._project.env` is internal to Architect and not documented, so best not to rely on it.

However, do we really need to read anything from the environment? You currently are reading the environment variables `LOCAL_OIDC_PORT` and `NODE_ENV`. However, I don't see a great need to support adjusting the port, and you don't really need to check `NODE_ENV` because the `sandbox` plugin hook will only be invoked when in sandbox mode anyway.